package.json

```diff
@@ -24,7 +24,9 @@
   "dependencies": {
     "bluebird": "^2.3.11",
     "coffee-script": "1.10.0",
-    "noflo": "0.5.21"
+    "noflo": "git+https://github.com/aretecode/noflo-built.git#experiment",
```

Comment:
@jonnor, not that I know of. It has been a month since it was released on npm and the official noflo github repo is not `built` so you can't use it in `package.json`. Additionally, https://github.com/noflo/noflo/pull/483 and https://github.com/noflo/noflo/pull/488 haven't been reviewed or merged.